consumatio/usecases/watch_time.py

```diff
@@ -0,0 +1,30 @@
+from consumatio.external.models import *
+from sqlalchemy import text
+
+
+class WatchTime:
+    def get_watch_time(self, tmdb, user, type):
+        runtime = 0
+        if type == "Movie":
```

Comment:
You could check for an invalid parameter and raise an InvalidParameter exception if necessary